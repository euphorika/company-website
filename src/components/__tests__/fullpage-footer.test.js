import React from "react"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

import FullpageFooter from "../fullpage-footer"

describe("FullpageFooter", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<FullpageFooter />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
