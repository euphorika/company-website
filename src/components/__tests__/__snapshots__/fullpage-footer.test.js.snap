// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullpageFooter renders correctly 1`] = `
<div
  className="fullpageFooter"
>
  <Footer />
</div>
`;
